docs/navigation-parameters-reference.md

```diff
@@ -1,294 +1,305 @@
+
+### skipRoutingForUrlPatterns
+- **type**: RegExp[]
+- **description**: defines regex patterns to be skipped by the router when listening for path changes. This property is used for excluding **redirect_uri** properties. Default patterns are `[/access_token=/, '/id_token=/]`.
+
+### pageNotFoundHandler
+- **type**: any
+- **description**: defines custom behavior when the 404 (page not found) error occurs.  Luigi handles it by default. Leave its body empty if you have an external 404 handling. You can return an Object with `redirectTo` property if you want Luigi to redirect to a specific navigation path after execution.|
+- **attributes**:
+  - **wrongPath** (string): the path that user tried navigating to
+  - **wasAnyPathFitted** (bool): it is true if Luigi managed to fit a valid path which means **wrongPath** was only partially wrong. Otherwise it is false.
+
+
+## Navigation properties
+The node navigation properties enable you to configure global navigation settings under the `navigation:` section in the configuration file.
+
+### nodeAccessibilityResolver
+- **type**: any
+- **description**: receives all values defined in the node configuration. It allows you to define a permission checker function that gets executed on every node. If it returns `false`, Luigi removes the node and its children from the navigation structure. See [angular navigation.js](../core/examples/luigi-sample-angular/src/luigi-config/extended/navigation.js) for an example.
+
+### defaults.isolateView
+- **type**: boolean
+- **description**: renders all views in new frames. This setting overrides the same-domain frame reuse. The **defaults.isolateView** is disabled by default, and you can overwrite it using the **isolateView** value on a single node level.
+
+### preloadViewGroups
+- **type**: boolean
+- **description**: allows deactivating the default preloading of [view groups](navigation-configuration.md#view-groups) iframes.
+
+### viewGroupsSettings
+- **type**: object
+- **description**: contains key-object pairs, where the key is the view group name as specified in the node properties, and the object contains key-value pairs. In each key-value pair, the key is the feature name and the value is the actual setting. The following options are supported:
+- **attributes**:
+  - **preloadUrl**(string): needs to be an absolute URL for a node from the view group. It is recommended that you use a dedicated small, visually empty view, which imports Luigi Client and is fine with getting an empty context, for example, without an access token. The **preloadUrl** property is also required for view group caching in case you need a view group iframe to refresh whenever you navigate back to it.
```

Comment:
"needs to be an absolute URL for a node from the view group" - I would rephrase it as the preload URL may not be an URL associated with any node. It is a view URL of a view belonging to the view group. It is a URL on the microfrontend side.